source/tools-and-extensions/archiving/HDB++.rst

```diff
@@ -41,7 +41,7 @@ Elements
 +--------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | HDB++ Diagnostic         | Standalone JAVA application that visualizes both the status of all the archiver device servers and the overall archiving system                                                                                                                                                                                                                                                                                                     |
 +--------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Archiving DB             | Specific Database devoted to storing attribute values. It can be either Mysql or Cassandra for the moment.                                                                                                                                                                                                                                                                                                                          |
+| Archiving DB             | Specific Database devoted to storing attribute values. The currently supported backend are Mysql, Cassandra or Timescaledb.                                                                                                                                                                                                                                                                                                         |
```

Comment:
Mysql, Cassanadra, Postgres or TimescaleDB

Reply:
Indeed postgres as well, as a collateral, but it works.

Reply:
Well, there's a native implementation for Postgres made by the guys in Russia...

Reply:
ok, cool, I didn't know about that !